package.json

```diff
@@ -41,11 +45,14 @@
       }
     ]
   },
-  "babel": {
-    "presets": [
-      [
-        "@babel/preset-env"
-      ]
-    ]
+  "devDependencies": {
+    "@babel/cli": "^7.10.5",
+    "@babel/core": "^7.11.1",
+    "@babel/preset-env": "^7.11.0",
+    "@beyondessential/eslint-config-beyondessential": "^2.5.0",
```

Comment:
Ooh this repo is still private, should we be very concerned? I guess it's only a dev dependency.

Reply:
I guess we should make this repo public too... It's a simple eslint config, plus it's publicly available in `npm`